netbox_netprod_importer/vendors/cisco/ios.py

```diff
@@ -78,3 +107,138 @@ def get_detailed_cdp_neighbours(self):
                 )
         for n in neighbours:
             yield n
+
+    def get_interface_mode(self, interface):
+        from pynxos.errors import CLIError
+        try:
+            return self._get_interfaces_mode()[
+                self.get_abrev_if(interface)].get("oper_mode")
+        except (KeyError, CLIError):
+            logger.debug("Switch %s, show interface switchport cmd error",
+                         self.device.hostname)
+        return None
+
+    def get_interface_access_valn(self, interface):
```

Comment:
Typo here, you probably meant `get_interface_access_vlan`

Reply:
Renamed